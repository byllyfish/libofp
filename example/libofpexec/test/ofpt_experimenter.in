{params: {type: OFPT_EXPERIMENTER, version: 1, msg: {experimenter: 0x01020304, exp_type: 0x10203040, data: '0123456789'}}, method: ofp.send, id: 1}
{params: {type: OFPT_EXPERIMENTER, version: 4, msg: {experimenter: 0x01020304, exp_type: 0x10203040, data: '0123456789'}}, method: ofp.send, id: 2}
