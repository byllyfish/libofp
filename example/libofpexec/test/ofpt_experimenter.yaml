---
type:            OFPT_EXPERIMENTER
xid:             0x00000000
version:         0x01
msg:             
  experimenter:    0x01020304
  exp_type:        0x00000000
  data:            0123456789
...
---
type:            OFPT_EXPERIMENTER
xid:             0x00000000
version:         0x04
msg:             
  experimenter:    0x01020304
  exp_type:        0x10203040
  data:            0123456789
...
