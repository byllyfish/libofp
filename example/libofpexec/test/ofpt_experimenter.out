{"id":1,"result":{"conn_id":0,"data":"0104001100000000010203040123456789"}}
{"id":2,"result":{"conn_id":0,"data":"040400150000000001020304102030400123456789"}}
